demo/apisix/tmail-apisix-plugin-runner/src/main/resources/application.yaml

```diff
@@ -18,6 +18,8 @@ redis:
   url: ${REDIS_URL:localhost:6379}
   password: ${REDIS_PASSWORD:}
   cluster.enable: ${REDIS_CLUSTER_ENABLE:false}
+  ignoreErrors: ${REDIS_IGNORE_ERRORS:true}
```

Comment:
BTW the default to `true` here would override the default to `false` at https://github.com/linagora/tmail-backend/pull/1025/files#diff-ef496fe2b5b9078292629685c38f588304ca2f4cbbc06df75f9ecfcd908cd4e7R34, no?